bundle/deploy/terraform/convert.go

```diff
@@ -150,6 +150,18 @@ func BundleToTerraform(config *config.Root) *schema.Root {
 		}
 	}
 
+	for k, src := range config.Resources.ModelServing {
+		var dst schema.ResourceModelServing
+		conv(src, &dst)
+		tfroot.Resource.ModelServing[k] = &dst
+
+		// Configure permissions for this resource.
+		if rp := convPermissions(src.Permissions); rp != nil {
+			rp.ServingEndpointId = fmt.Sprintf("${databricks_model_serving.%s.id}", k)
```

Comment:
```suggestion
			rp.ServingEndpointId = fmt.Sprintf("${databricks_model_serving.%s. serving_endpoint_id}", k)
```

We should use `serving_endpoint_id` to set permissions. Source: https://github.com/databricks/terraform-provider-databricks/blob/master/docs/resources/permissions.md#model-serving-usage

Reply:
Hmm that still didn't work
<img width="439" alt="Screenshot 2023-08-30 at 11 10 23 AM" src="https://github.com/databricks/cli/assets/87999496/1a654db2-2ad8-4f04-a70d-10d6c155f7f2">
<img width="656" alt="Screenshot 2023-08-30 at 11 10 38 AM" src="https://github.com/databricks/cli/assets/87999496/8f1295e8-9472-4cbf-ba8f-518cf511f982">

Reply:
You can find the generated terraform configuration inside the `.databricks` directory, in a file called `terraform/bundle.tf.json`. Looking at that should help. Happy to TAL.

Reply:
Looks like the underlying Terraform resource is broken since it's not setting the `serving_endpoint_id` field:

<img width="700" alt="Screenshot 2023-08-30 at 5 33 24 PM" src="https://github.com/databricks/cli/assets/87999496/e8f42dd3-74c3-4c13-8057-d44d88eb7c0e">
<img width="1038" alt="Screenshot 2023-08-30 at 5 33 51 PM" src="https://github.com/databricks/cli/assets/87999496/e6aead2d-77b2-4c8a-80a6-1139a5140830">

Which is weird since we seem to be explicitly setting it here: https://github.com/databricks/terraform-provider-databricks/blob/master/serving/resource_model_serving.go#L63

Reply:
Figured it out, looks like `bundle` is using an older version of the Terraform provider (`1.11.1`) that didn't have the permissions resource set-up for `model_serving`
<img width="422" alt="Screenshot 2023-08-30 at 5 48 39 PM" src="https://github.com/databricks/cli/assets/87999496/c2f862b4-0b9e-43ef-88dd-5cf4390282d5">

Oddly enough, when I went to go test this out locally and gave no version constraints, Terraform installed the same version and claimed it to be the latest version of the provider
<img width="415" alt="Screenshot 2023-08-30 at 5 51 22 PM" src="https://github.com/databricks/cli/assets/87999496/19e9b093-5604-43ac-8bbb-a4fc803ba701">
@nfx @alexott is this expected? Shouldn't the latest version be `1.24.1`?

Reply:
Please double check if you have a `~/.terraformrc` file. If you did any work on the provider in the past, it is possible you're still using your local build everywhere. If you remove that file, or remove the build from `~/.terraform.d`, it will pick up the latest version from the registry.

Reply:
Yeah I had checked for `~/.terraformrc` and confirmed that didn't exist, but turns out I did have a `~/.terraform.d`. Removing that and changing this line seems to have fixed it!